UnityProject/Assets/Scripts/Messages/Server/SetActionUIMessage.cs

```diff
@@ -183,6 +186,12 @@ private static NetMessage _Send(
 			// SpellList singleton index
 			else if (action is Spell spellAction)
 			{
+				if (spellAction.SpellData.Index == -1)
+				{
+					Loggy.LogError(
```

Comment:
Use correct logging format!
Also is their is an error, shouldn't this return and not attempt o continue execution?

Reply:
this was one of bods changes so ehh?

Reply:
I think it's fine since it's meant to Show up in the editor